### PR TITLE
add feature rclone mount

### PR DESCRIPTION
**Pull request by babidi34**

### CODE REVIEW

**Comment by stefangweichinger** [comment]
@babidi34 Thanks for the PR, will look into it in more detail asap.
Would you mind adding some description or some lines of HOWTO for the main README? Thanks.

File: tasks/mount.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    state: present
+
+- name: Link between /usr/bin/fusermount and /usr/bin/fusermount3
+  ansible.builtin.file:
+    src: /usr/bin/fusermount
+    dest: /usr/bin/fusermount3
+    state: link
+    force: yes
+
+- name: Create and active services rclone mount
```
Comment (stefangweichinger) [inline tasks/mount.yml:21]: I assume that should be "activate"

**Comment by babidi34** [comment]
> @babidi34 Thanks for the PR, will look into it in more detail asap. Would you mind adding some description or some lines of HOWTO for the main README? Thanks.

Thanks for the feedback! I've added the README update.

**Comment by stefangweichinger** [comment]
some linting issues ... @babidi34 would you have a look?

**Comment by babidi34** [comment]
yes I saw I will try to correct my PR this weekend

**Comment by stefangweichinger** [comment]
> yes I saw I will try to correct my PR this weekend

Fixed most of the issues locally, but can't push to your branch ... sure.

The missing issues:

```
 ansible-lint .
WARNING  Listing 3 violation(s) that are fatal
command-instead-of-module: systemctl used in place of systemd module
handlers/main.yml:6 Task/Handler: Reload and restart rclone-mount services

no-changed-when: Commands should not change things if nothing needs doing.
handlers/main.yml:6 Task/Handler: Reload and restart rclone-mount services

no-changed-when: Commands should not change things if nothing needs doing.
tasks/mount.yml:3 Task/Handler: Create directory backup on the cloud
```

I might look closer asap, but this week is quite busy already. And I'd prefer state-of-the-art ansible code ;-)

**Comment by stefangweichinger** [comment]
> I fixed lint error, normaly it's ok now

linting seems ok now, thanks. Patches look good at a first glance! nice work

**Comment by babidi34** [comment]
Thanks

**Comment by stefangweichinger** [comment]
@babidi34 pls give me some time for the merge. This will trigger a minor release update, and I always struggle with getting the tags right. The automatism doesn't work for me.

File: handlers/main.yml [hunk 2]
```diff
@@ -4,9 +4,5 @@
...
 
 - name: Reload and restart rclone-mount services
-  ansible.builtin.shell: |
-    systemctl daemon-reload
-    for service in /etc/systemd/system/rclone-mount@*.service; do
-      systemctl enable $(basename $service)
-      systemctl restart $(basename $service)
-    done
\ No newline at end of file
+  ansible.builtin.systemd:
```
Comment (stefangweichinger) [inline handlers/main.yml:7]: You know that this does not enable and restart the services? It's not the same behavior.

**Comment by stefangweichinger** [comment]
@babidi34 Your PR was merged and is in the release 0.1.5 here:

https://galaxy.ansible.com/ui/standalone/roles/stefangweichinger/ansible_rclone/versions/

thanks for your contribution!

**Comment by babidi34** [comment]
Thank you for creating this role. It simplifies my life in managing backups of my machines.

**Comment by stefangweichinger** [comment]
> Thank you for creating this role. It simplifies my life in managing backups of my machines.

You're welcome. It started as a small learning project ...

**Pull request merged**